x-pack/plugins/security_solution/public/management/pages/mocks/exceptions_list_http_mocks.ts

```diff
@@ -0,0 +1,259 @@
+/*
```

Comment:
An exceptions list API HTTP set of mocks that currently uses the `ExceptionsListItemGenerator`. I will look to improve this in the future and add support allowing a custom generator to be passed in (ex. TrustedApps, EventFilters, etc)